velox/functions/prestosql/window/tests/WindowTestBase.cpp

```diff
@@ -92,19 +96,29 @@ std::vector<RowVectorPtr> WindowTestBase::makeFuzzVectors(
 void WindowTestBase::testWindowFunction(
     const std::vector<RowVectorPtr>& input,
     const std::string& function,
-    const std::string& overClause) {
-  auto queryInfo = buildWindowQuery(input, function, overClause);
+    const std::string& overClause,
+    const std::optional<std::string> frameClause) {
+  auto queryInfo = buildWindowQuery(input, function, overClause, frameClause);
   SCOPED_TRACE(queryInfo.functionSql);
   assertQuery(queryInfo.planNode, queryInfo.querySql);
 }
 
 void WindowTestBase::testWindowFunction(
     const std::vector<RowVectorPtr>& input,
     const std::string& function,
-    const std::vector<std::string>& overClauses) {
+    const std::vector<std::string>& overClauses,
+    const std::optional<std::vector<std::string>>& frameClauses) {
   createDuckDbTable(input);
-  for (const auto& overClause : overClauses) {
-    testWindowFunction(input, function, overClause);
+  if (frameClauses) {
```

Comment:
On second thoughts it might better to use a default parameter {""} and write the loops where 

for every overclause
   for every frameclause

That would look cleaner.